runtime/ftplugin/man.vim

```diff
@@ -143,32 +138,32 @@ func <SID>GetPage(...)
       endwhile
     endif
     if &filetype != "man"
-      new
-      setl nonu fdc=0
+      tabnew
+      setlocal nonumber foldcolumn=0
     endif
   endif
   silent exec "edit $HOME/".page.".".sect."~"
   " Avoid warning for editing the dummy file twice
-  setl buftype=nofile noswapfile
+  setlocal buftype=nofile noswapfile
 
-  setl ma nonu nornu nofen
-  silent exec "norm 1GdG"
+  setlocal modifiable nonumber norelativenumber nofoldenable
+  silent norm! 1GdG
```

Comment:
Am 2015-07-01 09:44, schrieb Lucas Hoffmann:
> In runtime/ftplugin/man.vim [1]:
> 
>> 
>> - setl ma nonu nornu nofen
>> - silent exec "norm 1GdG"
>> + setlocal modifiable nonumber norelativenumber nofoldenable
>> + silent norm! 1GdG
> 
> Why not simply %delete? Why silent, can this produce an error?

It prevents the "No Lines in buffer" message